### ISSUE TITLE
Not playing nicely with SN30 pro+ (registration results in 50+ profiles?)

### ISSUE THREAD

**Issue by jose1711** [kozec/sc-controller#611]
Version 0.4.8.2, clean ssc profile, Arch Linux x86_64

With 8BitDo SN30 pro+ controller connected via USB:
1. Settings - Controllers - Register New Controller
2. SN30 pro+ identifies as "Microsoft X-Box 380 pad"
3. Test of input works
4. After clicking Save the system becomes unresponsive for several seconds

Then I get this:

![obrázok](https://user-images.githubusercontent.com/1406222/103276210-3cf56700-49c6-11eb-8926-c7102c28dd7a.png)

On terminal I see a stream of messages as follows:
```
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
W App           Daemon reported unknown profile: 'None'; Overriding.
```

**Comment by nephitejnf** [comment]
Does the same with the N30 Pro.

**Comment by jose1711** [comment]
Sorry, this is actually an issue for Ryochan7's fork (https://github.com/Ryochan7/sc-controller). @nephitejnf Could you perhaps confirm you're on Arch Linux (which would explain why you're seeing the same issue)?

**Comment by nephitejnf** [comment]
Yeah, installed from the AUR on artix (basically arch minus systemd).

**Comment by jose1711** [comment]
@kozec Is there a chance to transfer this issue to Ryochan7's fork please or I need to recreate it there? Thank you.

**Comment by IcedQuinn** [comment]
Also happens with the Pro 2.

If I switch the controller to DInput mode then the problem goes away. However it gets somewhat a woozy mapping because D-Input treats the left and right bumpers as a single axis that goes from negative to positive based on which bumper is held.
